packages/modular-views.macro/src/index.macro.js

```diff
@@ -56,20 +56,23 @@ const viewsMap = `({
 })`;
 
 function views({ references, babel }) {
-  if (references.default.length > 1) {
-    throw new Error(
-      'Do not use modular-views.macro more than once in a module, it will bloat your bundle size.',
-    );
-    // TODO: would be nice if we could restrict it to only one usage per _program_.
+  if (!references.default || references.default.length === 0) {
+    return;
   }
 
-  references.default.forEach((ref) => ref.replaceWithSourceString(viewsMap));
+  references.default[0].hub.file.path.node.body.unshift(
```

Comment:
this should probably be inserted after all import statements, not the very top like here.

Reply:
Ok I checked, and it's not a syntax error if this is at the very top, and it'll compile such that import statements get hoisted anyway. Leaving as is.